### ISSUE TITLE
WebView findText throws a primitive error

### ISSUE THREAD

**Issue by davidgranstrom** [supercollider/supercollider#4933]
## Environment

* SuperCollider version: 3.11.0 (develop-1a728802c)
* Operating system: macOS 10.14.6
* Other details (Qt version, audio driver, etc.): Qt 5.14.1 (homebrew)

## Steps to reproduce

Calling the `findText` method triggers the error. The same error can be reproduced by opening a HelpBrowser instance and using the "Find the text in document" utility (Ctrl-f). 

```supercollider
(
w = Window();
v = WebView(w, Rect(0,0,w.bounds.width,w.bounds.height));
v.findText("foo");
)
```

## Expected vs. actual behavior

```
ERROR: Primitive '_QObject_InvokeMethod' failed.
Failed.
RECEIVER:
Instance of WebView {    (0x119a304d8, gc=74, fmt=00, flg=00, set=06)
  instance variables [49]
    qObject : RawPointer 0x7fdc0a0a3450
    finalizer : instance of Finalizer (0x11bafd6b8, size=2, set=1)
    virtualSlots : nil
    wasRemoved : false
    font : nil
    resize : Integer 1
    alpha : Float 1.000000   00000000 3FF00000
    decorator : nil
    layout : nil
    userCanClose : true
    deleteOnClose : true
    action : nil
    mouseDownAction : nil
    mouseUpAction : nil
    mouseEnterAction : nil
    mouseLeaveAction : nil
    mouseMoveAction : nil
    mouseOverAction : nil
    mouseWheelAction : nil
    keyDownAction : nil
    keyUpAction : nil
    keyModifiersChangedAction : nil
    keyTyped : nil
    focusGainedAction : nil
    focusLostAction : nil
    dragLabel : nil
    beginDragAction : nil
    canReceiveDragHandler : nil
    receiveDragHandler : nil
    toFrontAction : nil
    endFrontAction : nil
    onClose : nil
    onResize : nil
    onMove : nil
    onLoadFinished : nil
    onLoadFailed : nil
    onLoadProgress : nil
    onLoadStarted : nil
    onLinkActivated : nil
    onLinkHovered : nil
    onReloadTriggered : nil
    onJavaScriptMsg : nil
    onSelectionChanged : nil
    onTitleChanged : nil
    onUrlChanged : nil
    onScrollPositionChanged : nil
    onContentsSizeChanged : nil
    onAudioMutedChanged : nil
    onRecentlyAudibleChanged : nil
}
PATH: /private/tmp/foo.scd
CALL STACK:
	MethodError:reportError
		arg this = <instance of PrimitiveFailedError>
	Nil:handleError
		arg this = nil
		arg error = <instance of PrimitiveFailedError>
	Thread:handleError
		arg this = <instance of Thread>
		arg error = <instance of PrimitiveFailedError>
	Object:throw
		arg this = <instance of PrimitiveFailedError>
	Object:primitiveFailed
		arg this = <instance of WebView>
	WebView:findText
		arg this = <instance of WebView>
		arg text = "foo"
		arg reversed = false
		arg func = <instance of Function>
	Interpreter:interpretPrintCmdLine
		arg this = <instance of Interpreter>
		var res = nil
		var func = <instance of Function>
		var code = "(
w = Window();
v = WebView(..."
		var doc = <instance of Document>
		var ideClass = nil
	Process:interpretPrintCmdLine
		arg this = <instance of Main>
^^ The preceding error dump is for ERROR: Primitive '_QObject_InvokeMethod' failed.
Failed.
RECEIVER: a WebView
```

**Comment by brianlheim** [comment]
@davidgranstrom thanks! this was an issue with a couple WebView methods; i've fixed it in #4937, give it a whirl if you have time. :)

**Comment by davidgranstrom** [comment]
@brianlheim Okay! Just tested with #4937 and it is working great in the HelpBrowser! My example code above also works after adding some html content:

```supercollider
(
w = Window();
v = WebView(w, Rect(0,0,w.bounds.width,w.bounds.height));
v.setHtml("<div>hello foo</div>");
v.findText("foo");
w.front;
)
```

Without any html content `findText` seem to hang sclang, but this feels more like a user error maybe? Just wanted to give you all info after my test in any case!

**Comment by brianlheim** [comment]
thanks! i'm not too sure about the other issues with findText, i also ran into them. based on what i'm seeing it is a mistake in qt docs or just that we are misusing the underlying calls. but i think those are minor issues compared to being unable to run the method at all.

**Comment by davidgranstrom** [comment]
@brianlheim Yes, I agree 👍

**Comment by brianlheim** [comment]
solved for 3.11.1, thanks for reporting @davidgranstrom !
